rollup.config.js

```diff
@@ -33,6 +34,7 @@ export default [{
 	    //   exclude: 'node_modules/**', // Don't transpile node_modules
 	    // }),
 		// commonjs(),
```

Comment:
@nswamy14  was this commented accidentally? Or now there is no dependency of commonjs?